src/components/Drawer/mdc-temporary-drawer.vue

```diff
@@ -0,0 +1,114 @@
+<!-- thanks to petejohanson for this implementation -->
+
+<template>
+  <aside class="mdc-temporary-drawer mdc-typography" :class="classes">
+    <nav ref="drawer" class="mdc-temporary-drawer__drawer">
+
+      <div class="mdc-temporary-drawer__toolbar-spacer" v-if="$slots['toolbar-spacer'] || toolbarSpacer">
+        <slot name="toolbar-spacer"/>
+      </div>
+
+      <header class="mdc-temporary-drawer__header" v-if="$slots.header">
+        <slot name="header"/>
+      </header>
+
+      <slot/>
+    </nav>
+  </aside>
+</template>
+
+<script lang="babel">
+  import { MDCTemporaryDrawerFoundation } from '@material/drawer'
```

Comment:
I think if we import like this:
`import MDCTemporaryDrawerFoundation from '@material/drawer/mdc_xxxx_foundation'`
webpack  won't pull the full MDC component and only the foundation dependencies.
I'm not 100% sure though, I'm pretty new with webpack and vue_loader